src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL8Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/ticket_app</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">password</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.model.Movie"/>
+        <mapping class="mate.academy.model.MovieSession"/>
+        <mapping class="mate.academy.model.CinemaHall"/>
+    </session-factory>
+</hibernate-configuration>
```

Comment:
```suggestion
</hibernate-configuration>

```
to avoid such symbols in your PR, make sure each file ends with exactly one empty line
![image](https://github.com/mate-academy/hibernate-create-movie-session-hw/assets/49767049/7d8a6179-f4c9-4361-b96e-3b3946aa52da)